chemprop/data/scaffold.py

```diff
@@ -71,8 +71,8 @@ def scaffold_split(data: MoleculeDataset,
              validation, and test splits of the data.
     """
     #TODO(degraff): floating point math could mess this up
```

Comment:
Really minor nitpick here, I suppose we don't need this TODO comment anymore since the latest commit addresses it.